contracts/core/funds-distribution-token/v1/interfaces/IBasicFDT.sol

```diff
@@ -5,16 +5,44 @@ import "lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 
 import "./IBaseFDT.sol";
 
+/// @title BasicFDT implements the basic level FDT functionality for accounting for revenues.
 interface IBasicFDT is IBaseFDT, IERC20 {
```

Comment:
Should we consolidate IBaseFDT and IBasicFDT?

Reply:
Will try.